components/KitBackgroundScreen.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { View, StyleSheet, Image, TouchableOpacity } from "react-native";
+import Colors from "../constants/Colors";
+import KitText from "./KitText";
+
+export default function KitBackgroundScreen(props) {
+  return (
+    <View style={styles.background}>
+      <View style={styles.main}>
+        <View>
+          <KitText style={styles.title} size={26}>
+            {props.title}
+          </KitText>
+          <TouchableOpacity
+            style={styles.backButton}
+            onPress={props.onPressBack}
+          >
+            <Image source={require("../assets/images/arrow.png")} />
+          </TouchableOpacity>
+        </View>
+        <View style={styles.container}>{props.children}</View>
+      </View>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  background: {
+    flex: 1,
+    backgroundColor: Colors.KIT_GREEN,
+    padding: 20
+  },
+  main: {
+    flex: 1,
+    borderRadius: 30,
+    backgroundColor: Colors.KIT_WHITE
+  },
+  container: {
+    padding: 16,
+    flex: 1
+  },
+  backButton: {
+    position: "absolute",
```

Comment:
Is absolute positioning the only way to do this?

Reply:
It's not the only way but I feel like it's the most correct way. Like we always want the button to be upper left no matter what and not care what the other components are doing.

Reply:
Discussed IRL, good to go